### ISSUE TITLE
relative path support?

### ISSUE THREAD

**Issue by Nuclearfossil** [mriehl/sublime_pybuilder#4]
As a user, I'd like to be able to define a path relative to my sublime-project file.

eg:

in ~/Code/research/test01/test01.sublime-project, I'd like to define the following:

``` bash
{
    "folders":
    [
        {
            "path": "."
        }
    ],
    "settings":
    {
        "python_interpreter": "./venv/bin/python",
        "project_root": "./",
        "extra_paths": ["./src/main/python"]
    }
}
```

Or some similar mechanism, so that I can deploy the sublime-project to other developers.

**Comment by mriehl** [comment]
Hi,

As far as I know there is no plugin API support for "what is the path of the current project file" in sublime (not even in ST3).
I think I remember seeing a hack in some plugins where a file is parsed to find out the path. I'll try to see if I can find out where it was.

Maybe using an environment variable would be helpful to you in the meantime? Like setting PROJECT_HOME=`pwd` and then using that environment variable in the sublime project file?
It's not possible right now but would be much easier to add than something that is relative to the sublime-project file.

**Comment by Nuclearfossil** [comment]
Yeah, I would expect that parsing the path from the project file name would
be the correct solution here in this case. Setting the PROJECT_HOME env
variable is a less preferential alternative.

-Ash
